compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/interop/LocalLayerContainer.uikit.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.interop
+
+import androidx.compose.runtime.compositionLocalOf
+import platform.UIKit.UIView
+
+internal val LocalLayerContainer = compositionLocalOf<UIView> {
```

Comment:
static?

Reply:
Done.
Also @igordmn maybe it's a good suggestion for Desktop.

On Desktop we have the same `internal val LocalLayerContainer = compositionLocalOf<Container>`.
Maybe for better performance, use staticCompositionLocalOf on Desktop too.

Reply:
Yes, static is better here